lib/rules/selector-attribute-operator-blacklist/index.js

```diff
@@ -37,7 +37,7 @@ const rule = function(blacklistInput) {
 
       parseSelector(rule.selector, result, rule, selectorTree => {
         selectorTree.walkAttributes(attributeNode => {
-          const operator = attributeNode.operator;
+          const operator = attributeNode.operator.trim();
```

Comment:
What if there no `attributeNode.operator`? `.trim()` will throw an error. In the next condition, there is a check `!operator`.

Reply:
Yes, you're right. I'll hold off making the change [until we know](https://github.com/stylelint/stylelint/issues/2757#issuecomment-333786942) if this is something we should be addressing in stylelint.